lib/batterylib/battery_oled.cpp

```diff
@@ -51,7 +61,11 @@ void Battery_OLED::update() {
     digitalWrite(ADC_EN, HIGH);
     delay(10);  // suggested by @ygator user in issue #2
     uint16_t v = analogRead(ADC_PIN);
+    #ifdef TTGO_T7
+       curv = ((float)v / 4095.0) * 7.58;
+    #else
```

Comment:
Because the 2 resistor that give the voltage to the ADC are different. With T7 v1.3 the are no voltage divider, and it is made by external resistors. The maximun voltage to ADC input is  about 1.1V.
The T7 v1.5, has an internal divider and the maximun voltage to the ADV is about 3V , I can not remember it exactly.